requirements.txt

```diff
@@ -1,10 +1,14 @@
+cloudpickle
+colorcet
 dask >= 0.15.4
 datashape >= 0.5.1
 numba >= 0.35.0
 numpy >= 1.7
 pandas >= 0.20.3
+param >=1.5.0,<2.0
 pillow
+scikit-image
 toolz
 xarray >= 0.9.6
-colorcet
-param >=1.5.0,<2.0
+
+pyspark>=2.0
```

Comment:
I don't want to require users to install pyspark, so this would somehow have to be an optional dependency.